### PR TITLE
Dark & Light toggle switch button 

### PR DESCRIPTION
**Pull request by TwoThreeTwo**
#254 

Things added:

1.-Toggle switch
2.-Full compatability across all devices (Hopefully, it works with my login screen lol, https://www.github.com/twothreetwo/website-login-screen)
3.-JQuery & Script to allow the transiton between Dark and Light to occur (I had to get a lot of help lol, might be bad code but it works :) )
4.- Made code scrumptious not delicious c :

Note from me:
Sorry for making the CSS Code well over 300 lines, it just works like that lol
but, this projec ultimately won't blind you cuz u can switch between dark and light which is pretty cool, it also uses transition speeds to make the process **super sexy**.

Here's the video providing you a preview of what it actually does :P

https://streamable.com/as86t5 
Any questions reach me on Two#6969 :)

### CODE REVIEW

**Comment by TwoThreeTwo** [comment]
@3arthqu4ke , idk if there's any issues with it, should work but please check it with ur device to see if it actually works, afaik it works with chrome on 1920x1080 resolution laptop

File: script.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+});
+setTimeout(function() {
+  $(".load-flash").css("display","none");
+  $(".load-flash").css("visibility","hidden");
+  tempDisableAnim();
+}, 20);
+$(window).resize(function() {
+  tempDisableAnim();
+  setTimeout(function() {
+    tempDisableAnim();
```
Comment (3arthqu4ke) [inline script.js:62]: Dont you need to define this function somewhere?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+function labelLight() {
+  $(".toggle-switch").attr("alt", "Go dark");
+  $(".toggle-switch").attr("title", "Go dark");
+}
+function goLight() {
+  console.log("Light mode is active");
+  labelLight();
+  $("body").removeClass("dark");
+  refreshFavicon();
```
Comment (3arthqu4ke) [inline script.js:27]: Shouldnt this function be defined somewhere?

**Review by 3arthqu4ke — CHANGES_REQUESTED** [review]

**Comment by 3arthqu4ke** [comment]
This is very cool, thanks for contributing. For some reason the Light - Dark button does not work on mobile for me. It works on my desktop with Chrome Devtools to see the mobile page tho 🤔

**Comment by TwoThreeTwo** [comment]
I'll fix up the things u requested when I'm not in school or not busy, might be a while

**Comment by mdo992** [comment]
sometimes i can’t understand people’s motives

**Comment by TwoThreeTwo** [comment]
> This is very cool, thanks for contributing. For some reason the Light - Dark button does not work on mobile for me. It works on my desktop with Chrome Devtools to see the mobile page tho 🤔

Could be due to different web browsers, like Chrome works on laptop but idk it works with safari or Opera, also the throttling could effect it somehow, but I'll do some debugging in a bit most likely will be ready by next week but, I may have to close this PR because it'll take a bit b4 I can actually have time to do anything Web Development related, school has started and I have too much homework lol

**Comment by TwoThreeTwo** [comment]
Also, most of this other stuff was included in an old project of mine which was the first time I tried using JS, the ppl who helped me a bit did send some code to fix some issues but ig it included some other code. I can probably just spend a good while clearing the useless js

**Comment by TwoThreeTwo** [comment]
> sometimes i can’t understand people’s motives

I just get bored

**Comment by 3arthqu4ke** [comment]
Its fine, its fine! Just leave this PR open. Take as much time as you want maybe my phone is just trash.

**Comment by TwoThreeTwo** [comment]
update on this, I'm just gonna rewrite it with a more simplistic basic bitch design with minimal js and css lol, I'll do it in a bit

**Comment by TwoThreeTwo** [comment]
This'll be just a simple little basic btn with 4 lines of js instead of 200 lines lol

**Comment by TwoThreeTwo** [comment]
@3arthqu4ke just changed up the design with a good ol' Click Me button made with js.

Reason why I built it with js? just easier lol, i watched 5 minutes of a js video and created that, the function I just had helped with a friend, overall should work WITH LITERALLY FUCKING ANYTHING.

**Comment by TwoThreeTwo** [comment]
Updated the js code to just refer to button.style = `foo` because it looks cleaner also i just discovered it lol

**Comment by 3arthqu4ke** [comment]
Uhm, whenever I press space it toggles dark/light mode. And the button at the bottom says "Click Me"?

**Comment by 3arthqu4ke** [comment]
unknown_2022 09 30-22 38

Every jump it switches

**Comment by mdo992** [comment]
> Uhm, whenever I press space it toggles dark/light mode. And the button at the bottom says "Click Me"

you could probably click away from the button and it’ll stop but the issue probably is that it’s still selected after you click it. @TwoThreeTwo

**Comment by TwoThreeTwo** [comment]
> unknown_2022 09 30-22 38
> 
> 
> 
> Every jump it switches
> 
> 
I'm jus gonna close this and reopen it later if I ever try to fix it

**Pull request closed without merging**